style.css

```diff
@@ -0,0 +1,69 @@
+body {
```

Comment:
For this file, lets put it in the css folder and rename it settings_modal.css I think (make sure to update the link tag in the html as well :smiley: )